tests/ruby/RuboCopBearTest.py

```diff
@@ -1,57 +1,91 @@
 import os
+from queue import Queue
+
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
 
 from bears.ruby.RuboCopBear import RuboCopBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
 
-good_file = """def good_name
```

Comment:
Actually, this is a linter specific problem. This is the first linter that I've come across that is careful about the filename extension. Moreover, it would make less sense IMO to improve `LocalBearTestHelper` to take care of this case only. Also we had to somehow improve the tests and use `check_results(...)` instead of `verify_local_bear(...)`.